.github/ISSUE_TEMPLATE/1-O3R-ISSUE-FORM.yaml

```diff
@@ -0,0 +1,87 @@
+name: Bug Report on @o3r
+description: File a bug report on @o3r scope
+title: "[Bug]: "
+labels: ["bug", "o3r"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure we have all the information we need to investigate efficiently.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: package-name
+    attributes:
+      label: Package name
+      description: Which package are you facing issue with?
+      multiple: false
```

Comment:
It could be linked to multiple packages no ?

Reply:
I was hesitating, but you are right it makes sense, I will change it. Thanks